docs/guides/create_service_existing_agent.md

```diff
@@ -1 +1,212 @@
-This tutorial will be added soon.
\ No newline at end of file
+The typical use case that this guide intends to illustrate is where a service owner wants to use already developed agents to run its own service, possibly customizing it through the service parameters. For example, an oracle agent service with existing agents, where the service owner specifies
```

Comment:
...where a service owner wants to use *existing* agents to run their own service *with custom parameters*. For example, an oracle *service customized to serve data of their interest*.

Reply:
parameters -> configuration